contents/blog/posthog-vs-amplitude.md

```diff
@@ -0,0 +1,420 @@
+        	<td></td>
+        	<td></td>
+    	</tr>
+    	<tr>
+			<td>Open source</td>
+        	<td className="text-center"><span className="text-red text-lg">✖</span></td>
+        	<td className="text-center"><span className="text-green text-lg">✔</span></td>
+        	<td className="text-center"><span className="text-green text-lg">✔</span></td>
+    	</tr>
+    	<tr>
+        	<td>Transparent pricing</td>
+        	<td className="text-center"><span className="text-red text-lg">✖</span></td>
+        	<td className="text-center"><span className="text-green text-lg">✔</span></td>
+        	<td className="text-center"><span className="text-green text-lg">✔</span></td>
+    	</tr>
+    	<tr>
+       	<td>100% self-serve</td>
+        	<td className="text-center"><span className="text-red text-lg">✖</span></td>
+       	    <td className="text-center"><span className="text-green text-lg">✔</span></td>
+        	<td className="text-center"><span className="text-green text-lg">✔</span></td>
+    	</tr>
+    	<tr>
+        	<td>1st-party cookies</td>
+        	<td className="text-center"><span className="text-red text-lg">✖</span></td>
+        	<td className="text-center"><span className="text-red text-lg">✖</span></td>
+        	<td className="text-center"><span className="text-green text-lg">✔</span></td>
+   		</tr>
+    	<tr>
+        	<td>Custom plugins</td>
+        	<td className="text-center"><span className="text-red text-lg">✖</span></td>
+        	<td className="text-center"><span className="text-red text-lg">✖</span></td>
+       	<td className="text-center"><span className="text-green text-lg">✔</span></td>
+    	</tr>
+    	<tr>
+        	<td>Multi-site</td>
+        	<td className="text-center"><span className="text-green text-lg">✔</span></td>
+        	<td className="text-center"><span className="text-green text-lg">✔</span></td>
+        	<td className="text-center"><span className="text-green text-lg">✔</span></td>
+    	</tr>
+    	<tr>
+        	<td>Unlimited users</td>
+        	<td className="text-center"><span className="text-green text-lg">✔</span></td>
+        	<td className="text-center"><span className="text-green text-lg">✔</span></td>
+        	<td className="text-center"><span className="text-green text-lg">✔</span></td>
+    	</tr>
+    	<tr>
+        	<td>Autocapture</td>
+        	<td className="text-center"><span className="text-red text-lg">✖</span></td>
+        	<td className="text-center"><span className="text-green text-lg">✔</span></td>
+        	<td className="text-center"><span className="text-green text-lg">✔</span></td>
+    	</tr>
+		<tr>
+        	<td>GDPR compliance</td>
+        	<td className="text-center"><span className="text-green text-lg">✔</span></td>
+        	<td className="text-center"><span className="text-red text-lg">✖</span></td>
+        	<td className="text-center"><span className="text-green text-lg">✔</span></td>
+    	</tr>
+		<tr>
+        	<td>HIPAA compliance</td>
```

Comment:
I mean, we do claim this already afaik @charlescook-ph?

Reply:
I think we can say yes because other solutions don't actually enable you to take that step, whereas we do (so I'm happy for us to claim we are HIPAA compliant)